types/to-absolute-glob/to-absolute-glob-tests.ts

```diff
@@ -0,0 +1,6 @@
+import * as toAbsoluteGlob from 'to-absolute-glob';
```

Comment:
Since `toAbsoluteGlob` is a function here, it should be `import ... = require(...)` here